pkg/util/workerqueue/workerqueue.go

```diff
@@ -150,8 +151,14 @@ func (wq *WorkerQueue) processNextWorkItem() bool {
 	}
 
 	if err := wq.SyncHandler(key); err != nil {
+		// Conflicts are expected, so only show them in debug operations.
+		if k8serror.IsConflict(err) {
+			wq.logger.Debug(err)
```

Comment:
Should this also use the WithField to include the keyName in the debug log?

Reply:
That's a good idea. I'll make that change 👍